experimental/AlgebraicStatistics/src/DiscreteGraphicalModels.jl

```diff
@@ -0,0 +1,86 @@
+function bron_kerbosch(G, R, P, X)
+
+    cliques = []
+
+    if length(P) == 0 && length(X) == 0
+        
+        push!(cliques, R)
+    end
+
+    Q = P
+    Y = X
+
+    for v in Q
+
+        new_R = vcat(R, v);
+        new_Q = [p for p in Q if p in neighbors(G, v)]
+        new_Y = [p for p in Y if p in neighbors(G, v)]
+
+        cliques = vcat(cliques, bron_kerbosch(G, new_R, new_Q, new_Y))
```

Comment:
I've removed this file for now since it is still far from ready and wasn't supposed to be in this PR but we'll definitely make these changes when we add it back in later.